patches/chrome-browser-ui-layout_constants.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/ui/layout_constants.cc b/chrome/browser/ui/layout_constants.cc
+index d73e202ea44900b5074112f93b11145eec8ecfc8..c1b18f9ee8f54ba2ca77c141cb44d4314f7d437e 100644
+--- a/chrome/browser/ui/layout_constants.cc
++++ b/chrome/browser/ui/layout_constants.cc
+@@ -16,6 +16,7 @@
+ 
+ int GetLayoutConstant(LayoutConstant constant) {
+   const bool touch_ui = ui::TouchUiController::Get()->touch_ui();
++  BRAVE_GET_LAYOUT_CONSTANT
```

Comment:
`#define LayoutConstant LayoutConstant constant) {
   const absl::optional ....
   return GetLayouyConstant_ChromiumImpl(constant); 
 }
 int GetLayouyConstant_ChromiumImpl(LayoutConstant` ?

Reply:
Changed in https://github.com/brave/brave-core/pull/19398/commits/552611182df14bfb4541da65c27e893fb3d747e0